StantDrawing/BrushTool/BrushToolView.swift

```diff
@@ -126,40 +137,63 @@ public class BrushToolView: UIView {
     }
     
     private func loadGroupPencilSizeButtons() {
-        let btnSize1   = Button(imageName: "", imageColor: blackHexColor, selector: #selector(self.pencilSize))
-        let btnSize2   = Button(imageName: "", imageColor: blackHexColor, selector: #selector(self.pencilSize))
-        let btnSize3   = Button(imageName: "", imageColor: blackHexColor, selector: #selector(self.pencilSize))
-        let btnSize4   = Button(imageName: "", imageColor: blackHexColor, selector: #selector(self.pencilSize))
-        let btnSize5   = Button(imageName: "", imageColor: blackHexColor, selector: #selector(self.pencilSize))
-        
-        btnSize1.uiButton.tag = 5
-        btnSize2.uiButton.tag = 10
-        btnSize3.uiButton.tag = 30
-        btnSize4.uiButton.tag = 55
-        btnSize5.uiButton.tag = 80
-        
-        btnSize1.uiButton.frame.size = CGSize(width: 18, height: 18)
-        btnSize2.uiButton.frame.size = CGSize(width: 23, height: 23)
-        btnSize3.uiButton.frame.size = CGSize(width: 28, height: 28)
-        btnSize4.uiButton.frame.size = CGSize(width: 33, height: 33)
-        btnSize5.uiButton.frame.size = CGSize(width: 37, height: 37)
+        let btnSize1   = Button(imageName: "pencilSize1", imageColor: blackHexColor, selector: #selector(self.pencilSize))
```

Comment:
I know that wasn't your task, but try to rename for button ;)

Reply:
Changed :)